### ISSUE TITLE
Keys Backup can be setup twice

### ISSUE THREAD

**Issue by bmarty** [vector-im/riot-android#2973]
While setup banner is displayed on Home, if I create the backup on another client, and then click on the banner, I will create a new backup.

When clicking on the setup banner, a force refresh should be done (going to manage setup Activity?)

**Comment by bmarty** [comment]
I assume there is no proper way to fix this, because backup can also be created on web when we are on the screen to create a backup.

**Comment by giomfo** [comment]
I disagree, we have to handle at least the case where the user is not setting up simultaneously a backup on another device.
I confirm "When clicking on the setup banner, a force refresh should be done" thx
